ArtOfIllusion/src/artofillusion/material/ProceduralMaterial3D.java

```diff
@@ -361,8 +324,8 @@ public void acceptEdits(ProcedureEditor editor)
   {
     initThreadLocal();
     int i = editor.getScene().indexOf(this);
-    if (i > -1)
-      editor.getScene().changeMaterial(i);
+    if(i == -1) return;
```

Comment:
This is not a good idea. I know that you do not like the single-statement stye:
```
if(cond)
   doThis();
```
But the inverted condition with an early return is worse. Please __do not__ do this. The style in the block above is correct.

As a second point, style changes outside of the methods that you are editing __Should Not__ be done, as the make the diff harder to understand. If something is seriously difficult to read/needs a re-format, please open a separate issue to discuss.